packages/workspace/src/utils/rules/to-js.ts

```diff
@@ -1,23 +1,58 @@
 import { transpile, JsxEmit, ScriptTarget } from 'typescript';
-import { forEach, Rule, when } from '@angular-devkit/schematics';
+import {
+  forEach,
+  when,
+  chain,
+  CreateFileAction,
+  Rule,
+} from '@angular-devkit/schematics';
 import { normalize } from '@angular-devkit/core';
 
-export function toJS(): Rule {
-  return forEach(
-    when(
-      (path) => path.endsWith('.ts') || path.endsWith('.tsx'),
-      (entry) => {
-        const original = entry.content.toString('utf-8');
-        const result = transpile(original, {
-          allowJs: true,
-          jsx: JsxEmit.Preserve,
-          target: ScriptTarget.ESNext,
-        });
-        return {
-          content: Buffer.from(result, 'utf-8'),
-          path: normalize(entry.path.replace(/\.tsx?$/, '.js')),
-        };
-      }
-    )
+const changeFileExtensions = forEach(
+  when(
+    (path) => path.endsWith('.ts') || path.endsWith('.tsx'),
+    (entry) => {
+      const original = entry.content.toString('utf-8');
+      const result = transpile(original, {
+        allowJs: true,
+        jsx: JsxEmit.Preserve,
+        target: ScriptTarget.ESNext,
+      });
+      return {
+        content: Buffer.from(result, 'utf-8'),
+        path: normalize(entry.path.replace(/\.tsx?$/, '.js')),
+      };
+    }
+  )
+);
+
+const changeExtensionsInConfigFiles: Rule = (tree, context) => {
+  const files = new Set(
+    tree.actions
+      .filter((action) => action.kind === 'c')
```

Comment:
This will affect any "created" file.  Not sure if we need to narrow that scope or not.

Reply:
Makes sense.